### DISCUSSION TITLE
Prisma Error In Deployment

### DISCUSSION THREAD

**Discussion by BishoyNarouz** [prisma/prisma#6585] [Q&A] [answered]
![Screenshot from 2021-04-15 01-09-31](https://user-images.githubusercontent.com/40390342/114791953-b05d9180-9d87-11eb-9e72-74670aa92f43.png)

i got this error in every time i make deploy in digital ocean, and the code works will in local 
how can i solve this error??

**Comment by ryands17** [comment, marked as answer]
Try running `prisma generate` before `nest build` on Digital Ocean. This should work.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
